lib/admin_page.php

```diff
@@ -49,17 +51,17 @@ function ppi_render_admin_page() {
         && $mysqlCompatible
         && $hostingCompatible;
 
-    if (! $isCompatible || ! ppi_p7_is_installed()) {
+    if (! $isCompatible || ! p7_is_installed()) {
```

Comment:
I would vote that version names are removed from functions as well - since the namespace includes the version